Directory.Build.targets

```diff
@@ -9,6 +9,25 @@
     <ImportDirectoryBuildTargets>false</ImportDirectoryBuildTargets>
   </PropertyGroup>
 
+  <!-- temporarily remap AssemblyKey to StrongNameKeyId -->
```

Comment:
Given there aren't many keys in this repo you might consider replacing them now.

Reply:
What does that entail?

Reply:
Just find/replace AssemblyKey property with the mapped StrongNameKeyId property.

Reply:
Ah, gotcha. Done.